TTBot/DataAccess/ChampionshipResults.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Dapper;
+using ServiceStack.Data;
+using ServiceStack.OrmLite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TTBot.Models;
+using TTBot.Services;
+
+namespace TTBot.DataAccess
+{
+    public class ChampionshipResults : IChampionshipResults
+    {
+        private readonly IDbConnectionFactory _conFactory;
+
+        public ChampionshipResults(IDbConnectionFactory conFactory)
+        {
+            _conFactory = conFactory;
+        }
+
+        public async Task AddAsync(List<ChampionshipResultsModel> championshipResults)
+        {
+            using (var connection = _conFactory.Open())
+            {
+                foreach (ChampionshipResultsModel resultsModel in championshipResults)
+                {
+                    await connection.InsertAsync(new ChampionshipResultsModel()
+                    {
+                        EventId = resultsModel.EventId,
+                        Driver = resultsModel.Driver,
+                        Points = resultsModel.Points,
+                        Positions = resultsModel.Positions
+                    });
+                }
+            }
+        }
+
+        public async Task DeleteAllAsync<ChampionshipResulstModel>()
```

Comment:
I can't see any usages of this but the delete should be specific to the Guild that the event is linked to, rather than totally clearing the DB.

Reply:
There's a typo in the type. It should be being used by https://github.com/alex9158/TTBot/pull/6/files#diff-06dd4fb87cd3d46433462ff4527400028532b4a24782c0c9c89d551222907044R57. We have no update functionality, so we clear down all results and read them in from the workbook. This allows us to correct previous race results.